ghost/core/core/server/models/role-utils.js

```
@@ -0,0 +1,41 @@
+// check if the user has an assigned role
+// so that we can stop writing this everywhere:
+//_.some(loadedPermissions.user.roles, {name: 'Administrator'})
+
+const _ = require('lodash');
+
+function checkUserPermissionsForRole(loadedPermissions, roleName) {
+    console.log('received permissions: ', loadedPermissions);
```

ESLint

[error] Unexpected console statement.

(no-console)